rw_tlv.m4

```diff
@@ -355,131 +355,71 @@ m4_define(['m4_into_fields'], ['{m4_into_fields_lhs(['$2'], [''], $1_FIELDS)['}
 
 // Reduction macro.
 // Performs an operation across all instances of a hierarchy and provides the result outside that hierarchy.
-// m4+redux($sum[7:0], >hier, max, min, $addend, '0, +)
-m4_define_plus(['m4_redux'], ['
-
-m4_pushdef(['m4_redux_sig'],    ['$5'])
-m4_pushdef(['m4_hier'],         ['$6'])
-m4_pushdef(['m4_MAX'],          ['$7'])
-m4_pushdef(['m4_MIN'],          ['$8'])
-m4_pushdef(['m4_sig'],          ['$9'])
-m4_pushdef(['m4_init'],         ['$10'])
-m4_pushdef(['m4_op'],           ['$11'])
-
-'], m4___file__, m4___line__, ['
+// m4+redux($sum[7:0], /hier, max, min, $addend, '0, +)
+\TLV redux($_redux_sig,/_hier,#_MAX,#_MIN,$_sig,$_init,#_op)
    \always_comb
```

Comment:
It's a little funny to use $_init for something that can be an expression, but it's probably the best choice. There's a similar case below, called $_cond_expr. Let's make this one $_init_expr as an indication that an expression is okay.